docs/docs/deploying.md

```diff
@@ -8,8 +8,8 @@ There are three main environment variables you need to specify in your deploymen
 
 | Variable                    | Meaning                                                                 | Where to get                                                                                                                                                                     |
 | --------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `QUIRREL_TOKEN`             | access token for the Quirrel server.                                    | Create a new Project + Client in the [Dashboard](https://quirrel.dev/dashboard).                                                                                                 |
-| `QUIRREL_BASE_URL`          | The base URL of your deployment.                                        | You probably know this. Something like `my-application.com`.                                                                                                                     |
+| `QUIRREL_TOKEN`             | access token for the Quirrel server.                                    | **Hosted:** Create a new Project + Client in the [Dashboard](https://quirrel.dev/dashboard). **Self-Hosted:** Check [How to deploy our own server](#acquiring-a-token) .         |
+| `QUIRREL_BASE_URL`          | The base URL of your client's deployment.                               | You probably know this. Something like `my-application.com`.                                                                                                                     |
```

Comment:
```suggestion
| `QUIRREL_BASE_URL`          | The base URL of your application's deployment.                               | You probably know this. Something like `my-application.com`.                                                                                                                     |
```

I generally prefer speaking of "Application" instead of "Client". The "Client" is the piece of JavaScript that your "Application" uses to interface with the Quirrel API, but it's the "Application" that gets deployed.